src/Multipart/UploadState.php

```diff
@@ -76,6 +94,62 @@ public function setPartSize($partSize)
         $this->partSize = $partSize;
     }
 
+    /**
+     * Set displayProgress. Sends object size to setProgressThresholds.
+     *
+     * @param $totalSize numeric Size of object to upload.
+     */
+    public function setDisplayProgress($totalSize)
+    {
+        $this->setProgressThresholds($totalSize);
+        $this->displayProgress = true;
```

Comment:
Why is this set here?  If we have made it to this function, shouldn't it already be set?